book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_5_make_it_move.md

```diff
@@ -43,7 +43,7 @@ or by using `docker ps`.
 
 Use the following command to run the joystick demo:
 
-    laptop $ docker run -dit --privileged --name joystick --net host -v /data:/data duckietown/rpi-duckiebot-joystick-demo:master18
+    laptop $ docker -H ![Duckiebot name].local run -dit  --device /dev/input/js0 --device /dev/i2c-1 --name joystick --network=host -v /data:/data duckietown/rpi-duckiebot-joystick-demo:master18

```

Comment:
why --network=host here but --net host in other places is there a difference

Reply:
no, I'm changing the instances to --network=host because it's clearer